src/iiif/tools.py

```diff
@@ -170,18 +174,26 @@ def get_info_from_iiif_url(iiif_url, source_file):
         raise InvalidIIIFUrlError(f"Invalid iiif url: {iiif_url}")
 
 
-def img_is_public(metadata, document_barcode):
+def img_is_public_copyright(metadata, document_barcode):
+    """
+    Return if document is public and has copyright.
+    If it is not public the copyright is not used en returned as unknown
+    """
+    public = None
+    copyright1 = None
```

Comment:
is er een reden dat deze var copyright1 heet ipv copyright (zonder 1)?

Reply:
PyCharm  zegt dat copyright een reserved word is en geeft daar een warning voor

Reply:
Ah echt, nooit van gehoord. Maar prima.